sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -230,13 +246,15 @@ func runCtr(containerdSocket string, namespace string, containerID string, sourc
 				// Mount in the persistent storage location for this container
 				{
 					Options:     []string{"rbind", "rw"},
-					Destination: "/.bottlerocket/host-containers/" + containerID,
-					Source:      "/local/host-containers/" + containerID,
+					Destination: "/.bottlerocket/" + persistentPath,
+					Source:      "/local/" + persistentPath,
```

Comment:
Is there a particular reason bootstrap containers get a different on-disk location than regular host containers?

Reply:
I think the main reason is to prevent collisions if two containers have the same name